### DISCUSSION TITLE
Port 22 SSH

### DISCUSSION THREAD

**Discussion by ray-bun** [coollabsio/coolify#2466] [General]
Hi, 
What is port 22 use for?  

https://coolify.io/docs/knowledge-base/server/firewall

Can I disable it and only enable when I need to SSH into the server? Does coolify need port 22 for the deployment?

**Comment by w7tf** [comment]
When you have multiple servers, SSH is needed for your "localhost" server to connect to the other server. Usually you'd have coolify as a "controller" and one or multiple servers where things actually run on. 

If your controller has a dedicated IP, you could add a firewall to the other server to only accept requests from your coolifys controller IP on port 22 and deny all other requests.
